packages/babel-plugin-proposal-async-generator-functions/test/fixtures/yield-star-polyfill-corejs3/options.json

```diff
@@ -0,0 +1,10 @@
+{
+  "parserOpts": {
+    "allowReturnOutsideFunction": true
+  },
+  "plugins": [
+    "transform-async-to-generator",
+    "proposal-async-generator-functions",
+    "./inject-corejs-async-iterator-polyfill.js"
```

Comment:
Maybe we can add a `preloadScripts` option to the test runner instead?

```json
{
  "parserOpts": {
    "allowReturnOutsideFunction": true
  },
  "plugins": [
    "transform-async-to-generator",
    "proposal-async-generator-functions"
  ],
  "preloadScripts": [
    "core-js/es/symbol/async-iterator.js"
  ]
```

Reply:
If we don't want to do the plugin hack yeah that could be ok - we should probably document all the config changes somewhere (for ourselves)

Reply:
My plan at that time was to use `babel-plugin-polyfill-corejs3` but I forgot the error messages I came across.

Reply:
Probably it's that it wasn't supported by Node.js 6-8.
We can revisit this after Babel 8.

Reply:
In Babel 8 this polyfill won't be needed anyway, since `Symbol.asyncIterator` will be natively supported.